modules/apps/object/object-admin-rest-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,155 @@
+components:
+    schemas:
+        ObjectDefinition:
+            properties:
+                actions:
+                    additionalProperties:
+                        additionalProperties:
+                            type: string
+                        type: object
+                    readOnly: true
+                    type: object
+                dateCreated:
+                    format: date
+                    readOnly: true
+                    type: string
+                dateModified:
+                    format: date
+                    readOnly: true
+                    type: string
+                id:
+                    format: int64
+                    readOnly: true
+                    type: integer
+                name:
+                    type: string
+                objectFields:
+                    items:
+                        $ref: "#/components/schemas/ObjectField"
+                    type: array
+            type: object
+        ObjectField:
+            properties:
+                dateCreated:
+                    format: date
+                    readOnly: true
+                    type: string
+                dateModified:
+                    format: date
+                    readOnly: true
+                    type: string
+                id:
+                    format: int64
+                    readOnly: true
+                    type: integer
+                indexed:
+                    type: boolean
+                indexedAsKeyword:
+                    type: boolean
+                indexedLanguageId:
+                    type: string
+                name:
+                    type: string
+                type:
+                    type: string
+info:
+    license:
+        name: "Apache 2.0"
+        url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+    title: "Object"
+    version: v1.0
+openapi: 3.0.1
+paths:
+    "/object-definitions":
+        get:
+            operationId: getObjectDefinitionsPage
+            parameters:
+                - in: query
+                  name: page
+                  schema:
+                      type: integer
+                - in: query
+                  name: pageSize
+                  schema:
+                      type: integer
+                - in: header
```

Comment:
Why sometimes we add the Accept-Language header explicitly in the endpoint and sometimes don't?

Reply:
At the beginning, to avoid duplication following Brian's suggestion. But we added it back some time ago and we should continue expliciting all those parameters.